src/Parsing/Impl/Microsoft.Python.Parsing.csproj

```diff
@@ -35,6 +35,11 @@
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <None Update="Microsoft-Python-Parsing.nuspec">
```

Comment:
Microsoft.Python.Parsing.nuspec ?

Reply:
I think nuget does not tolerate dots.

Reply:
I just keep the nuspec name the same style as before.. but the nuget id is using ```.```